workshop/exercises/docker-compose.yml

```diff
@@ -41,34 +43,34 @@ services:
       - 5000:80
 
     volumes:
-      - ./pygeoapi/docker.config.yml:/pygeoapi/local.config.yml
+      - ./pygeoapi.config.yml:/pygeoapi/local.config.yml
      # - ./data:/data # Exercise 1 - Ready to pull data from here
-     # - ./pygeoapi/es-entrypoint.sh:/es-entrypoint.sh
-     # - ./pygeoapi/wait-for-elasticsearch.sh:/wait-for-elasticsearch.sh
+     # - ./elastic/es-entrypoint.sh:/es-entrypoint.sh
```

Comment:
Think all ES text should be moved to exercises/elastic and place a docker-compose.yml there. My suggestion is to remove all ES stuff here. Keep this compose file simple. ES will be advanced topic and we can refer to exercise/elastic then.